airflow/utils/db.py

```diff
@@ -1480,6 +1498,10 @@ def upgradedb(
 
     with create_global_lock(session=session, lock=DBLocks.MIGRATIONS):
         log.info("Creating tables")
+        if not to_revision and not _get_current_revision(session=session):
+            # New DB; initialize and exit
+            initdb(session=session)
```

Comment:
This is a slight change in behaviour: 

Previously if you ran `airflow db upgrade` you would never env up with default connections created. Now you might if the database doesn't currently exist.